misd/phone-number-bundle/1.3/config/packages/misd_phone_number.yaml

```diff
@@ -1,6 +1,6 @@
 # To persist libphonenumber\PhoneNumber objects, add the Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType mapping to your application's config.
 # This requires: doctrine/doctrine-bundle
-doctrine:
-    dbal:
-        types:
-            phone_number: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
+# doctrine:
```

Comment:
not related to that PR, but doctrine is not required dependency

Reply:
Yeah sure, but if you uncomment this configuration, it becomes one.